src/BenchmarksApps/TechEmpower/BlazorSSR/BlazorSSR.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Npgsql" Version="$(NpgsqlVersion80)" />
   </ItemGroup>
 
-  <ItemGroup Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '9.0'))">
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'net9.0'))">
```

Comment:
This fixes a build error that occurs when building the project without specifying a target framework, e.g. when building from VS :smile: